cms/envs/common.py

```diff
@@ -766,6 +766,10 @@
     'notes',
 ]
 
+# XBlock types listed here will _always_ be selectable as Studio components
+XBLOCKS_ALWAYS_IN_STUDIO = [
```

Comment:
It looks like we were each correct.
- We want the key set in common.py so that all other envs will inherit
  some default value (here it's `[]`).
- We also want the bit in aws.py so that it'll try to override the value
  with the entry in `cms.envs.json`.